website/package.json

```diff
@@ -8,11 +8,11 @@
     "deploy": "docusaurus deploy"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.54",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.54",
+    "@docusaurus/core": "^2.0.0-0",
+    "@docusaurus/preset-classic": "^2.0.0-0",
```

Comment:
This would match the highest currently available beta version (from lock file, 2.0.0-beta.4). Just pointing out, to be sure this is intended 😄

Reply:
Yeah, this is intentional 😄